src/Components/Mood/Mood.module.css

```diff
@@ -0,0 +1,83 @@
+.flexContainer {
```

Comment:
This name is a bit fishy since it isn't using `display: flex` nor is it really describing what it's doing